data.sql

```diff
@@ -1,10 +1,11 @@
 /* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
-
  INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(1,'Agumon','2020-02-03',0,True,10.23);
  INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(2,'Gabumon','2018-11-15',2,True,8.0);
  INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(3,'Pikachu','2021-01-07',1,False,15.04);
- INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(4,'Devimon','2017-05-12',5,True,11);
\ No newline at end of file
+ INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(4,'Devimon','2017-05-12',5,True,11);
+ INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(5,'Charmander','2020-02-08',0,False,-11.0);
+ INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(6,'Plantmon','2021-11-15',2,True,-5.7);
+ INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(7,'Squirtle','1993-04-02',3,False,-12.13);
+ INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(8,'Angemon','2005-06-12',1,True,-45.0);
+ INSERT INTO animal (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(9,'Blossom','1998-10-13',3,True,-17);
```

Comment:
- [ ]  Kindly add the following  data as it is on the project requirements: :+1: 

`Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.`

- [ ] In line 10 where Blossom weight should be `17kg` instead of `-17kg`. :+1: 

`Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.`